frontendGraphApi/build.gradle

```diff
@@ -0,0 +1,11 @@
+dependencies {
+  vavr()
```

Comment:
Can be moved out of the `dependencies` section

Reply:
Done, fixed on the follow-up PR